application/controllers/logic.py

```diff
@@ -8,38 +8,54 @@
 
 
 def deal_hands(age, players):
+    """Call to have the next age's hands dealt"""
+    if (age not in range(1, 4)) or (len(players) not in range(3, 8)):
+        print("Invalid call, age: " + age + " Number of players: " + len(players))
+        return False
+
     cards = Card.query.filter(Card.noPlayers <= len(players)).filter_by(age=age).all()
+    if len(cards) is not 7*len(players):
```

Comment:
I will take it out after we play an actual game - I just want to make sure that all the cards are actually there.